modules/machineset-yaml-nutanix.adoc

```diff
@@ -78,26 +78,31 @@ endif::infra[]
       providerSpec:
         value:
           apiVersion: machine.openshift.io/v1
-          cluster:
+          categories: <5>
+            key: <prism_category_name>
+            value: <prism_category_value>
```

Comment:
categories is a list of category objects, so something like this would be more apt
```
categories:
  - key: category_name
    value: category_value
```